### PR TITLE
Added a form in the new UI for Osquery

### PR DESCRIPTION
**Pull request by simstoykov**

### CODE REVIEW

File: grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_form.ng.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div [formGroup]="form" class="container">
+  <mat-form-field appearance="outline">
+    <mat-label>Query</mat-label>
+    <textarea matInput placeholder="" formControlName="query" cdkTextareaAutosize class="query"></textarea>
+  </mat-form-field>
+
+  <mat-form-field class="timeout">
+    <mat-label>Timeout</mat-label>
+    <input matInput placeholder="" formControlName="timeoutMillis">
```
Comment (max-vogler) [inline grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_form.ng.html:9]: Please limit the values of this input field to numbers.

**Review by max-vogler — CHANGES_REQUESTED** [review]
Please provide a cropped screenshot showing your UI changes.

**Comment by simstoykov** [comment]
Thanks for the review, that's how it looks: Osquery from

**Review by mbushkov — APPROVED** [review]

**Comment by simstoykov** [comment]
@max-vogler Can you PTAL when you have time (could be after tomorrow)

**Pull request merged**